Solutions/Digital Shadows/Analytic Rules/Digital_Shadows_incident_creation_exclude.yaml

```diff
@@ -0,0 +1,84 @@
+﻿id: f7abe9c1-1e6c-4317-b907-25769e7764c5
+name: Digital Shadows Incident Creation for exclude-app
+description: ''
+severity: Medium
+requiredDataConnectors:
+  - connectorId: DigitalShadows
+    dataTypes: 
+      - DigitalShadows_CL
+queryFrequency: 5m
+queryPeriod: 6m
+triggerOperator: gt
+triggerThreshold: 0
+tactics: []
```

Comment:
Please insert a relevantTechniques: field here as well.

Reply:
should it be empty if I don't have anything to put in it?
`relevantTechniques: []`
will this be correct?

Reply:
Please use relevantTechniques: []